suzieq/engines/rest/engineobj.py

```diff
@@ -74,7 +74,8 @@ def _get_response(self, verb: str, **kwargs) -> pd.DataFrame:
                                               self.iobj.namespace),
                       'hostname': kwargs.get('hostname', self.iobj.hostname)}
         kwargs.update(cmd_params)
-        kwargs = {k: v for k, v in kwargs.items() if v}
+        # Cannot drop values which are equal to 0 or False
+        kwargs = {k: v for k, v in kwargs.items() if v or v in [0, False]}
```

Comment:
Many of the values are lists aren't they? If so, would this still work?

Reply:
If I understood correctly, you are asking if empty lists are dropped. The answer is yes. The only difference is that now '0' and 'False' are no more dropped.